pullrequests/index.html

```diff
@@ -21,7 +21,7 @@ <h2>This is your first pull request</h2>
 	</ul>
 
 	<p>Below this p tag there is another p tag with the name and date of the last author. Change it for your own.</p>
-	<p>Cutiko, June 26 2019</p>
+	<p>Sam, July 29 2019</p>
```

Comment:
I'm reviewing just to review, so visitors can see what it's like. I added this comment by clicking on line 24.

Reply:
I'll just merge, but if you were to make edits due to a request in this comment, you'd then respond with something like, "Done", Fixed", or a question here, etc. Then, when it was done, you could click "Resolve conversation". I tend to resolve mine whenever I can on the Angular repo so people know I addressed the comment. It also neatly collapses them. 

The conversations here can prove super helpful later, especially for large, long-lived PRs.

Reply:
Awesome!